### PR TITLE
add Rocket.Chat

### PR DESCRIPTION
**Pull request by k0nsl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ A curated list of Minio community projects inspired by [awesome-go](https://gith
 * [QuezX](https://quezx.com/) - QuezX is a recruitment aggregator for connecting employers to recruitment agencies. It uses Minio to store and manage CV's and other documents on their platform.
 * [Trustvox](http://site.trustvox.com.br) - Trustvox is an integrator of ecommerce platforms, ensures 100% valid product reviews, allowing only real buyers to post reviews along with a continuous audit process. The platform team chose Minio to create its own S3 service to avoid infrastructure vendor lock-in, obtaining transparent compatibility with Google Cloud Storage and previous Amazon S3 infrastructures. We tested other options before, and only Minio showed the stable and enterprise-grade results we were looking for.
 * [www.houbank.com](https://www.houbank.com/home/) - Houbank uses Minio as a simple, reliable and compatible self-host AWS S3 alternative. Houbank uses Minio to upload user files and exchange documents in internal applications/jobs.
+* [Rocket.Chat](https://rocket.chat/) - Rocket.Chat is a completely FOSS alternative to Slack.
```
Comment (nitisht) [inline README.md]: Could you also add a line about, what is Minio used for in Rocket Chat?

Comment (k0nsl) [reply README.md]: Yes, excuse me for that oversight. I will do so.

**Review by k0nsl — COMMENTED** [review]
I hope this is enough.

**Comment by nitisht** [comment]
It is perfect. Thanks for the contribution @k0nsl

**Review by nitisht — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**